aws-lc-sys/include/rust_wrapper.h

```diff
@@ -20,9 +23,9 @@ extern "C" {
 // The following functions are wrappers over inline functions and macros in
 // BoringSSL, which bindgen cannot currently correctly bind. These wrappers
 // ensure changes to the functions remain in lockstep with the Rust versions.
-int ERR_GET_LIB_RUST(uint32_t packed_error);
-int ERR_GET_REASON_RUST(uint32_t packed_error);
-int ERR_GET_FUNC_RUST(uint32_t packed_error);
+AWS_LC_SYS_EXPORT int ERR_GET_LIB_RUST(uint32_t packed_error);
```

Comment:
Can we use `OPENSSL_EXPORT` from `"openssl/base.h"`? Seems like it would come in handy, so we wouldn't need duplicate definitions in both `rust_wrapper.h`s

Reply:
Is that intended for use outside the library? It might work if we define `BORINGSSL_IMPLEMENTATION` and `BORINGSSL_SHARED_LIBRARY` prior to its include.

Reply:
hmm I was under the impression that these were already defined if we built with AWS-LC. 
* https://github.com/aws/aws-lc/blob/eaa19c77d000175da90da7b4ef22ffe24dfc2a2f/INCORPORATING.md?plain=1#L72-L77

Not a blocker though, just curious if it was reusable